tests/test_managed_api.rs

```diff
@@ -0,0 +1,38 @@
+use openmls::prelude::*;
+
+mod utils;
+
+use utils::managed_utils::*;
+
+#[test]
+fn test_managed_api() {
+    // Some basic setup functions for the managed group.
+    let handshake_message_format = HandshakeMessageFormat::Plaintext;
+    let update_policy = UpdatePolicy::default();
+    let callbacks = default_callbacks::default_callbacks();
+    let managed_group_config =
+        ManagedGroupConfig::new(handshake_message_format, update_policy, callbacks);
+    let number_of_clients = 20;
+    let setup = ManagedTestSetup::new(managed_group_config, number_of_clients);
+    setup.create_clients();
+
+    for ciphersuite in Config::supported_ciphersuites() {
+        let group_id = setup.create_random_group(3, ciphersuite).unwrap();
+        let mut groups = setup.groups.borrow_mut();
+        let group = groups.get_mut(&group_id).unwrap();
+
+        // Add two new members.
+        let (_, adder_id) = group.members[0].clone();
+        let new_members = setup.random_new_members_for_group(group, 2).unwrap();
+        setup
+            .add_clients(ActionType::Commit, group, &adder_id, new_members)
+            .unwrap();
+
+        // Remove a member
+        let (_, remover_id) = group.members[2].clone();
+        let (_, target_id) = group.members[3].clone();
+        setup
+            .remove_clients(ActionType::Commit, group, &remover_id, vec![target_id])
+            .unwrap();
```

Comment:
Ok so this is how I use the framework to create a group and add/remove members to/from a group. But how do I test something. I guess we always want to verify the group state after an operation or at the end. Could you do this here so we know how to then write other tests?